src/commands/info/help.ts

```diff
@@ -38,6 +38,18 @@ export default class extends Command {
 				});
 			}
 
+			if (command.options) {
+				let concatOptions = '';
+				command.options.forEach(param => {
+					concatOptions += `**${param.name}** (${param.required ? 'required' : 'optional'}): ${param.description}\n`;
+				});
```

Comment:
This can be done with a .map() and a .join() if you want to avoid string concatenation and have it be a oneliner.